src/jit/codegencommon.cpp

```diff
@@ -7147,6 +7159,14 @@ void CodeGen::genProfilingLeaveCallback(unsigned helper /*= CORINFO_HELP_PROF_FC
 
 #elif defined(_TARGET_X86_)
 
+#if defined(UNIX_X86_ABI)
+    // CodeGen::genAlignStackBeforeCall
+    unsigned saveCntStackDepth      = getEmitter()->emitCntStackDepth;
+    getEmitter()->emitCntStackDepth = 0;
```

Comment:
This needs a comment (an explanation): why are we saving / restoring this, and setting it to zero here?